air_gapped/build_bk.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+# Build the docker image for the air gapped docs.
+
+set -eo pipefail
+
+export AIR_GAPPED=docker.elastic.co/docs-private/bk_air_gapped:latest
+
+cd $(git rev-parse --show-toplevel)
+
+if [[ ! -d /opt/git-mirrors/built-docs.git ]]; then
+  echo "/opt/git-mirrors/built-docs.git must exist and contain a reference clone of the built-docs repo"
+  exit 1
+fi
+
+# Get an up to date copy of the repo
+rm -rf air_gapped/work
```

Comment:
is this needed if the agent is clean?

Reply:
agree, I forgot that the agent is being cleaned